build.sbt

```diff
@@ -28,32 +30,36 @@ val sonatypeReleaseSettings = Seq(
   )
 )
 
-def projectWithPlayVersion(majorMinorVersion: String) =
-  Project(s"play-v$majorMinorVersion", file(s"play-v$majorMinorVersion")).settings(
-    scalaVersion       := "2.12.18",
-    crossScalaVersions := Seq(scalaVersion.value, "2.13.11"),
+def projectWithPlayVersion(playVersion: PlayVersion) =
+  Project(s"play-v${playVersion.majorMinorVersion}", file(s"play-v${playVersion.majorMinorVersion}")).settings(
+    crossScalaVersions := Seq(scalaVersion.value) ++ (if (playVersion.supportsScala3) Seq("3.3.1") else Seq.empty),
     scalacOptions ++= Seq("-feature", "-deprecation"),
+    Compile / unmanagedSourceDirectories += baseDirectory.value / playVersion.pekkoOrAkkaSrcFolder,
 
     libraryDependencies ++= Seq(
-      "com.gu.play-secret-rotation" %% "core" % "0.37",
+      "joda-time" % "joda-time" % "2.12.5",
```

Comment:
Given that Play 2.9 and 3.0 support Java 11+ only, you might be be able to remove joda-time time and replace it with `java.time.*`?

Reply:
Good idea! Yep, I've pulled that into its own PR, which has now been released as `play-googleauth` [v2.8.8](https://github.com/guardian/play-googleauth/releases/tag/v2.2.8).